app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,10 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="green">#00B300</color>
+    <color name="red">#FF0000</color>
```

Comment:
The name of the colors also has to be descriptive

Reply:
done